_tests/browser_tests/README.md

```diff
@@ -0,0 +1,90 @@
+## Quick start:
+
+From the project root:
+
+```sh
+# Update Selenium Server binaries:
+webdriver-manager update
+```
+
+From this directory:
+
+```sh
+# Start Selenium Server:
+webdriver-manager start
+
+# Open a new tab and tell protractor to start testing:
+protractor conf.js
+```
+
+If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
+
+```sh
+protractor conf.js --baseUrl=beta.consumerfinance.gov
```

Comment:
This line didn't work. But it did work when I changed it to `protractor conf.js --baseUrl=http://beta.consumerfinance.gov`